Gemfile

```diff
@@ -47,6 +47,7 @@ gem "toastr-rails"
 gem "actiontext", github: "kobaltz/actiontext", branch: "archive", require: "action_text"
 gem "webpacker", "> 4.0"
 gem 'sidekiq-scheduler'
+gem 'bootstrap-datepicker-rails'
```

Comment:
I don't *think* this is necessary -- you might need to update your local branch. This was added in #1248 (as you noted) but I think perhaps it wasn't merged when you started this branch?